### ISSUE TITLE
Shouldn't decoupler require seaborn as a dependency?

### ISSUE THREAD

**Issue by pcm32** [saezlab/decoupler-py#69]
On a conda installation I see on runtime:

```
File "/home/.../miniconda3/envs/__decoupler@1.4.0/lib/python3.11/site-packages/decoupler/plotting.py", line 26, in check_if_seaborn
    import seaborn as sns
ModuleNotFoundError: No module named 'seaborn'
```

any other missing dependency? I'm trying to fix the bioconda package.

Thanks!

**Comment by PauBadiaM** [comment]
Hi @pcm32,

Many thanks for doing this!!! Unfortunately I'm swarmed with work and didn't have the time to work on it.
Regarding your question, there are some dependencies that I do not enforce to install because they are not really required to run most of the code for `decoupler`.
Here's the list of potential dependencies that are not included in the `setup.py`:
```
skranger
scikit-learn
omnipath
pypath-omnipath
matplotlib
seaborn
adjustText
scanpy
```
Let me know if you need anything else

**Comment by PauBadiaM** [comment]
Solved in 5babf3db1e8cdf73cfadecc89613970b5462405e
